app/views/posts/show.html.slim

```diff
@@ -1,17 +1,41 @@
+= stylesheet_link_tag 'comments'
+
 .dflex.justify-content-center
   .card style="padding:2% 5% 2% 5%"
-    small.text-muted
-      | Posted by&#160
-      i
+    span
+      small.text-muted
+        | Posted by
         = User.find(@post.user_id).username
-      | &#160&#183
+      - if user_signed_in? && current_user == @post.user
```

Comment:
īstenībā, ja neesi logged in, tad current user ir nil un iekš rubu ir safe salīdzināt objektus vienu ar otru, pat ja tie ir atšķirīgas klases instances.
`nil == @post.user => false`